src/rpc/rawtransaction.cpp

```diff
@@ -839,3 +839,124 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
 
     return hashTx.GetHex();
 }
+
+UniValue verifyrawtransactions(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() < 1 || params.size() > 2)
+        throw runtime_error(
+            "verifyrawtransactions [\"hexstring\",...] ( options )\n"
+            "\nVerifies one or more raw transactions (serialized, hex-encoded). If transactions depend on each other, they must be provided in order.\n"
+            "\nArguments:\n"
+            "1. [\"hexstring\",...] (array of strings, required) The hex string of the raw transactions)\n"
+            "2. options   (json object, optional)\n"
+            "     {\n"
+            "       \"include_mempool\"          (boolean, optional, default=true) Whether to include the mem pool\n"
+            "       \"check_final\"              (boolean, optional, default=true) Check that the transactions will be final by next block\n"
+            "       \"check_standard\"           (boolean, optional, default=true) Perform transaction standard checks\n"
```

Comment:
As said I would leave this out for now, until we have more relay/mining policy encapsulated to expose here in a more generic way.